Add a basic python.cmd so users can run pepper from cmd.exe easily

jheiselman opened the pull request:
Resolves #73 by adding a basic batch script that gets installed to $PYTHON_HOME\scripts

Commit:
Add a basic python.cmd so users can run pepper from cmd.exe easily

gtmanfred commented on scripts/pepper.cmd:
```diff
@@ -0,0 +1,5 @@
+set _SCRIPTDIR=%~d0
+set _SCRIPTPATH=%~p0
+
+python "%_SCRIPTDIR%%_SCRIPTPATH%pepper" %*
```
Can you add a newline at the end of this file?

jheiselman replied:
I tried adding a newline, but it looks like Github is stripping it. As for the question of how this differs from adding the peppr script to bin/, this is for Windows users. The pepper script doesn't get added to any bin directory. Further, Windows users are used to being able to run commands without specifying the interpreter first and by default Python doesn't setup a file association that results in the execution of scripts, but in the editing of them.

gtmanfred replied:
looks like the newline is good now.

Ok, so I don't have to do anything else to let the script work?

jheiselman replied:
No. Maybe just make a note somewhere pointing Windows users to it. It will otherwise work out its own install location and properly call the pepper python script that is provided by this repo.

gtmanfred replied:
Can you make a note about it in the README?

gtmanfred reviewed (changes requested):
How does this get installed to the path differently from pepper binary being installed to bin/ ?

jheiselman committed:
Added newline at the EOF as requested for PR182

jheiselman committed:
Add mention of how to run pepper

Mention the pepper script location and the convenience script for Windows users

gtmanfred committed:
Merge branch 'develop' into win_pepper_cmd
